python-frontend/src/main/java/org/sonar/python/parser/PythonParser.java

```diff
@@ -38,11 +38,15 @@ public final class PythonParser {
   private final Parser<Grammar> sslrParser;
 
   public static PythonParser create() {
-    return new PythonParser();
+    return new PythonParser(new IPythonGrammarBuilder().create());
```

Comment:
Shouldn't we use `PythonGrammarBuilder` here instead?

Reply:
fixed